pages/ActivityStreams/Overview.md

```diff
@@ -33,22 +33,347 @@ These deviations from the standard will be thoroughly documented here.
+
+For example, the following would be a valid audio objects:
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Audio",
+  "url": "https://example.org/short_clip.webm"
+}
+```
+
+```json
+{
+  "@context": "https://www.w3.org/ns/activitystreams",
+  "type": "Audio",
+  "name": "Super Great Podcast",
+  "url": {
+    "type": "Link",
+    "href": "https://example.org/podcast.mp3",
+    "mediaType": "audio/mp3"
+  },
+  "preview": {
+    "type": "Link",
+    "href": "https://example.org/podcast_preview.mp3"
+  },
+  "duration": "PT3H12M"
+}
+```
+
+#### Image
+
+Clients are recommended to support presentation for activity objects of type image to represent image content posted by users.
+Image type objects, if supported, must at minimum contain a `"type"` field with the value `"Image"` and a `"URL"` with either a URL string or an activity sub-object of type link containing a URL pointing to an image file.
+
+Image objects are also recommended to include a `"mediaType"` field on their link sub-object with a standard [MIME type](https://www.iana.org/assignments/media-types/media-types.xhtml) string value.
+If no media type is provided, clients may attempt to infer the media type from the headers of the linked file or simply ignore the object.
+
+Clients that support image objects must at least support media types of `"image/jpeg"` complying with [RFC2045](https://www.iana.org/go/rfc2045), `"image/png"` complying with the [W3C PNG Standard](https://www.w3.org/TR/2003/REC-PNG-20031110/), `"image/svg+xml"` complying with the [W3C SVG standard](https://www.w3.org/Graphics/SVG/) and `"image/webp"` complying with the [WebP standard](https://developers.google.com/speed/webp/).
+Clients are also recommended to support `"image/gif"` complying with [RFC2045](https://www.iana.org/go/rfc2045) and `"image/heic"` complying with the [ISO/IEC JTC-1](http://www.iso.org/iso/jtc1_home.html) standard where possible.
+
+Image objects may include a `"preview"` field with an activity sub-object of type link, a URL string or array of sub-objects and/or strings referring to a smaller resolution image or images provided as previews of larger image files.
+Preview fields should generally include at least one file of the following resolutions: `320x240`, `640x480` or `1024x768`.
+
+Clients that support image objects should also include fields of `"height"` and `"width"` each with positive integer value either on the root activity object or the individual link sub-objects if multiple are present.
+If these fields are not provided, clients may attempt to infer the media type from the headers of the linked file or simply ignore the object.
+Clients may also ignore image files of exceptionally large or small sizes, however images with a preview of one of the prior mentioned resolutions must always be supported.
```

Comment:
My thinking was that this should be optional since height and width aren't always clearly defined in the case of unusual image formats like SVG or panoramic MP4.

Reply:
**Notes following meeting:**

- Height & Width must be included
- Remove preview recommendation entirely